courier/courier_dispatcher_test.go

```diff
@@ -37,11 +32,13 @@ func TestMessageTTL(t *testing.T) {
 		Body:    "test-body-1",
 	}))
 	require.NoError(t, err)
-	require.NotEqual(t, uuid.Nil, id)
+	require.NotZero(t, id)
```

Comment:
Did this assertion fail before? I think

```
require.NotEqual(t, uuid.Nil, id)
```

is clearer than `NotZero` because it is not very obvious to me how `NotZero` works under the hood.

Reply:
No, wasn't failing, you're right. I think we talked about that in the PATCH PR. I'll change it back, though, as in this case it is more obvious to compare it with `uuid.Nil`.